src/IdentityServer/Extensions/ValidatedAuthorizeRequestExtensions.cs

```diff
@@ -165,8 +165,9 @@ private static NameValueCollection ToOptimizedRawValues(this ValidatedAuthorizeR
             {
                 // https://openid.net/specs/openid-connect-core-1_0.html#JWTRequests 
                 // requires client id and response type to always be in URL
+                // REVIEW - Does this apply to pushed requests?
```

Comment:
OIDC requires that when request objects are used, the client pass the client id and response type, in order to make the request a valid oauth request. With PAR, I don't think we need to worry about that, since PAR is an OAuth specification, and the requests we're making are valid PAR requests. In fact, an example in RFC 9126 pushes a request object containing a response_type claim without duplicating the response type outside the request object. 

Since OIDC is asking for this duplication in other circumstances in  order to be a valid oauth request, and the relevant oauth specification doesn't require it in this circumstance, I'm leaving it out.